### PR TITLE
[EPMEDU-953] Create Thank you page

### PR DESCRIPTION
**Pull request by aklepets**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -54,6 +54,8 @@ android {
 }
 
 dependencies {
+    implementation(projects.feature.home)
```
Comment (Fileek) [inline app/build.gradle.kts]: Please delete it. Our app module should know only about host screens and splash screen at the moment.

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Thanks()
+        }
+        BackButton(onBackToHome)
+    }
+}
+
+@Composable
+fun Thanks() {
```
Comment (Fileek) [inline feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt]: Can you please rename it to ThankYouContent, move the composable to thankyou.ui package and make it internal to follow consistency in our project?

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        modifier = Modifier
+            .padding(24.dp)
+            .fillMaxWidth(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+    ) {
+        Image(
+            contentScale = ContentScale.None,
+            painter = painterResource(R.drawable.ic_paw),
+            contentDescription = null,
```
Comment (Fileek) [inline feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt:52]: Please replace null with some useful string

File: feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        )
+        Text(
+            text = stringResource(R.string.animals_fed),
+            style = MaterialTheme.typography.body2,
+        )
+    }
+}
+
+@Composable
+fun BackButton(onBackToHome: () -> Unit) {
```
Comment (Fileek) [inline feature/home/src/main/java/com/epmedu/animeal/home/presentation/thankyou/ThankYouScreenUi.kt]: Can you please do the same as with Thanks composable? You can just rename it to BackToHomeButton. And I suggest to rename parameter lambda to onClick.

**Review by Fileek — APPROVED** [review]

File: app/build.gradle.kts [hunk 5]
```diff
@@ -54,6 +54,7 @@ android {
 }
 
 dependencies {
+
```
Comment (Fileek) [inline app/build.gradle.kts]: You forgot to remove the blank line)

**Review by Makentoshe — APPROVED** [review]

**Review by carlossantiagomenendez — APPROVED** [review]

**Pull request merged**